roles/haproxy/tasks/main.yml

```diff
@@ -144,15 +155,36 @@
   notify:
     - "reload haproxy"
 
+- name: Copy haproxy acls
+  template:
+    src: "{{ item }}.j2"
+    dest: "/etc/haproxy/acls/{{ item }}"
+    owner: root
+    group: lbops
+    mode: 0664
+  with_items:
+    - validvhostsrestricted.acl
+    - validvhostsunrestricted.acl
+  notify:
+    - "reload haproxy"
+
+- name: Copy haproxy maps
+  template: 
+    src: "{{ item }}.j2"
+    dest: "/etc/haproxy/maps/{{ item }}"
+    owner: root
+    group: lbops
+    mode: 0664
+  with_items:
+    - backends.map
+    - redirects.map
+  notify:
+    - "reload haproxy"
+
 - name: Place file with list of browsers that do not accept samesite=none cookies
   copy:
     src: nosamesitebrowsers.lst
-    dest: /etc/haproxy/nosamesitebrowsers.lst
-    owner: root
-    group: root
-    mode: 0644
-  notify: 
-    - "reload haproxy" 
+    dest: /etc/haproxy/maps/nosamesitebrowsers.lst
```

Comment:
Why is this not part of the previous task? Only because it’s not a template? Because the permissions and reload behaviour now differ.

Reply:
Good catch, I've combined the two into one task.

Reply:
Moet je de file wel renamen naar nosamesitebrowsers.lst.j2

Reply:
Thanks for reminding me :)

I've made it into two tasks again, the build is now green, and the permissions are what they should be. 
Merging ..